modules/monitoring-using-node-selectors-to-move-monitoring-components.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: CONCEPT
+[id="using-node-selectors-to-move-monitoring-components_{context}"]
+= Using node selectors to move monitoring components
+
+By using the `nodeSelector` constraint with labeled nodes, you can move any of the monitoring stack components to specific nodes.
+By doing so, you can control the placement and distribution of the monitoring components across a cluster.
+
+By controlling placement and distribution of monitoring components, you can optimize system resource use, improve performance, and segregate workloads based on specific requirements or policies.
+
+[id="how-node-selectors-work-with-other-constraints_{context}"]
+== How node selectors work with other constraints
+
+
+If you move monitoring components by using node selector constraints, be aware that other constraints can exist for a cluster:
+
+* Topology spread constraints might exist to control pod scheduling within hierarchical network infrastructure topologies.
```

Comment:
we can be more generic, topologies are not necessary "network related": https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/ and https://docs.openshift.com/container-platform/4.12/nodes/scheduling/nodes-scheduler-pod-topology-spread-constraints.html

Reply:
apart from that, lgtm

Reply:
@machine424 has a point, though I understand we do refer to this use case in another place. Maybe we should relax this a bit? How about:
```suggestion
* Topology spread constraints might exist to control pod scheduling within a cluster's specific topology.
```
Otherwise we could also remove any ambiguity here:
```
* Topology spread constraints might exist.
```

Reply:
I updated the list item to make it more generic: `Topology spread constraints might be in place to control pod placement.`